explore/src/main/scala/explore/config/ConfigurationPanel.scala

```diff
@@ -46,13 +49,17 @@ final case class ConfigurationPanel(
 object ConfigurationPanel {
   type Props = ConfigurationPanel
 
-  implicit val propsReuse: Reusability[Props] = Reusability.derive
-  implicit val stateReuse: Reusability[State] = Reusability.never
+  implicit val propsReuse: Reusability[Props]                          = Reusability.derive
+  implicit val filterReuse: Reusability[AvailableFilter]               = Reusability.byEq
+  implicit val filterSetReuse: Reusability[SortedSet[AvailableFilter]] = Reusability.by(_.toSet)
+  implicit val optionsReuse: Reusability[ImagingConfigurationOptions]  = Reusability.derive
```

Comment:
Maybe this should go in `explore.model.reusability`.  Also, `lucuma.ui.reusability` can derive a `Reusability[SortedSet[A]]` from a `Reusability[A]`.